priam/src/main/java/com/netflix/priam/backup/BackupRestoreUtil.java

```diff
@@ -17,6 +17,7 @@
 
 package com.netflix.priam.backup;
 
+import com.google.api.client.util.Lists;
```

Comment:
I'm generally not a fan of using (non-shaded) guava in a library since it causes so many compatibility issues. I've gone as far as created my own `Preconditions` class. This usage below looks like it might be able to be replaced with a vanilla `ArrayList` constructor.

Reply:
Removed.